src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.hpp

```diff
@@ -75,7 +78,7 @@ namespace energyplus {
    * * If there is no cooling, no heating, no "both": None
    * * All other cases: "both"
    */
-  ENERGYPLUS_API ComponentType plantLoopType(const openstudio::model::PlantLoop& plantLoop);
+  OS_DEPRECATED ENERGYPLUS_API openstudio::energyplus::ComponentType plantLoopType(const openstudio::model::PlantLoop& plantLoop);
```

Comment:
Same here. I chose to use Loop_Impl::componentType to match HVACComponent_Impl, and note Loop_Impl::loopType. Maybe that was a dumb idea, don't know